### PR TITLE
test: actually use masternode with basic bls pubkey in mnauth test

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
`rpc_manuth` is failing in ~50% cases locally because we still use legacy pubkeys (not in 100% cases because sometimes they look like basic ones). In CI it fails too but we retry failed tests a few times so it's less noticeable. Example of "unlucky" tests: https://gitlab.com/dashpay/dash/-/jobs/8613271300#L1867.

#6467 follow-up

## What was done?
Add another masternode after v19 activaition to actually use basic bls pubkey

## How Has This Been Tested?
run tests

## Breaking Changes
n/a

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by PastaPastaPasta — APPROVED** [review]
makes sense, and change looks right.

utACK 3db20e3ed4070c1291c28868f90d283a088dd2b0

File: test/functional/rpc_mnauth.py [hunk 1]
```diff
@@ -21,8 +21,9 @@ def set_test_params(self):
 
     def run_test(self):
         self.activate_v19(expected_activation_height=900)
+        self.dynamically_add_masternode()
```
Comment (knst) [inline test/functional/rpc_mnauth.py:24]: I believe we don't need 2 of them:
```diff
-self.set_dash_test_params(2, 1)
+self.set_dash_test_params(1, 0)
```

**Review by knst — APPROVED** [review]
utACK ec00c372c7c4230fb7de75bb1a00d4b81896dcd6

**Review by PastaPastaPasta — APPROVED** [review]
utACK ec00c372c7c4230fb7de75bb1a00d4b81896dcd6

**Pull request merged**